dubbo-remoting/dubbo-remoting-http12/src/main/java/org/apache/dubbo/remoting/http12/HttpUtils.java

```diff
@@ -126,7 +126,7 @@ public static float parseQuality(String expr, int index) {
     public static List<Locale> parseAcceptLanguage(String header) {
         List<Item<Locale>> locales = new ArrayList<>();
```

Comment:
Can I move this line to line 131? This will avoid creating unnecessary object.

Reply:
OK